src/tools/registerOpenApi.ts

```diff
@@ -194,7 +194,7 @@ function buildToolCallback({
     }
 
     const response = await fetch(request);
-    const data = await response.json();
+    const data = await decodeResponse(response);
```

Comment:
I thought about it again, and I think we should just use `response.text()` no matter what.
There's no point in using `response.json()` when we're serializing it back into a `string` not even 10L later...
This will also fix your jsonl problem 👍